publishers/apiconnect/apic-backstage/1.0.0/package.yaml

```
@@ -0,0 +1,5 @@
+global:
+  dynamic:
+    plugins:
+      - package: oci://ghcr.io/redhat-developer/rhdh-plugin-export-overlays/apic-backstage:pr_1233__1.0.0!apic-backstage
```

Hi @johns-thomas,

the `pr_1233__1.0.0` tag is a container tag that is primarly build to verify a PR on the https://github.com/redhat-developer/rhdh-plugin-export-overlays repository.

I think you should use this oci uri instead:

```suggestion
      - package: oci://ghcr.io/redhat-developer/rhdh-plugin-export-overlays/apic-backstage:next__1.0.0!apic-backstage
```

I've asked internally if we could have a `latest` or `rhdh-1.7__latest` tag. /cc @davidfestal @kadel 

That image is build after your PR got merged. You can see all versions here https://github.com/orgs/redhat-developer/packages?q=theme&tab=packages&q=apic-backstage => https://github.com/redhat-developer/rhdh-plugin-export-overlays/pkgs/container/rhdh-plugin-export-overlays%2Fapic-backstage